### DISCUSSION TITLE
Class "Mpdf\Mpdf" not found

### DISCUSSION THREAD

**Discussion by OsamhAlahdl** [mpdf/mpdf#1812] [Q&A]
Fatal error: Uncaught Error: Class "Mpdf\Mpdf" not found in C:\xampp\htdocs\a\a.php:5 Stack trace: #0 {main} thrown in C:\xampp\htdocs\a\a.php on line 5

**Comment by ssieb** [comment]
You have to provide some actual information if you want a useful answer.

**Comment by shakyjake** [comment]
Sounds like your version of mPDF is out of date. Composer only installs the latest version that's compatible with your local PHP installation. Try using `composer require mpdf/mpdf:8.1.3` and it should tell you what you need, could be that you need to update PHP or (hopefully) just that you need to enable an extension (like `gd`).
